pmm-app/src/pmm-qan/panel/components/Filters/components/CheckboxGroup/CheckboxGroup.styles.ts

```diff
@@ -37,26 +36,39 @@ export const getStyles = stylesFactory((theme: GrafanaTheme) => {
       }
     `,
     filterHeaderWrapper: css`
-    display: flex;
-    justify-items: space-between;
-    margin-bottom: 0 !important;
-    margin-top: 20px !important;
-  `,
+      display: flex;
+      justify-items: space-between;
+      margin-bottom: 0 !important;
+      margin-top: 20px !important;
```

Comment:
nit: a lot of spacings here can be improved by using the theme

Reply:
Agree, I will do it when will work on filters refactoring, don't want to spend time for it now because it's not a high priority